packages/features/sales-escrows/modals/StartEscrowModal.tsx

```diff
@@ -103,10 +112,16 @@ export function StartEscrowModal({
 
   const onAmountChanged = (enteredAmount: string) => {
     setEnteredAmount(enteredAmount);
+    const fee = toLargerUnit(token.fee, token.decimals);
+    const amount = toBigNumber(enteredAmount);
+    const amountWithFee = amount.plus(fee);
+    const balance = toBigNumber(tokens[token.symbol].balance);
 
     let validationMsg = validateTokenAmount(enteredAmount, token.decimals);
     if (validationMsg) {
       setFormErrors({ ...formErrors, amount: validationMsg });
+    } else if (amountWithFee > balance) {
```

Comment:
```suggestion
    } else if (toBigNumber(enteredAmount).plus(fee).plus(fee).isGreaterThan(balance)) {
```
Also, has balance already been converted to the larger unit?
See: https://mikemcl.github.io/bignumber.js/#gt